samples/test-samples.cmake

```diff
@@ -40,14 +40,12 @@ endif ()
 
 # Set SAMPLES_LIST so that helloworld becomes the first if BUILD_ENCLAVES=ON.
 if (BUILD_ENCLAVES)
-  set(SAMPLES_LIST helloworld file-encryptor switchless host_verify)
```

Comment:
Why was host_verify removed here?

Reply:
Sorry to make you confused. There is a PR https://github.com/openenclave/openenclave/pull/3463 which may remove it before merging.

Reply:
https://github.com/openenclave/openenclave/blob/master/docs/Community/Contributing.md says:

> **DON'T** mix independent, unrelated changes in one PR.

Reply:
Removed, thanks!